vagga.yaml

```diff
@@ -99,7 +99,7 @@ commands:
     description: Run tests
     container: test
     run: python3 manage.py test
-  pre-commit: !Command
```

Comment:
Why did you discontinue this command? It's useful for manual runs of certain checks